src/uniprotkb/components/results/UniProtKBCard.tsx

```diff
@@ -34,11 +34,12 @@ const UniProtKBCard = ({ data }: { data: UniProtkbAPIModel }) => {
       'Disease',
     ]);
 
-    return categorisedKeywords.map((keywordCategory, index) => (
-      <Fragment key={keywordCategory.category}>
-        {index > 0 && ' · '}
-        <KeywordList keywords={keywordCategory.keywords} inline />
-      </Fragment>
+    return categorisedKeywords.map((keywordCategory) => (
```

Comment:
<img width="2160" alt="Screenshot 2021-12-01 at 14 28 42" src="https://user-images.githubusercontent.com/4357962/144252852-a58bb2ad-ada0-4ee7-9c35-697b51282c3d.png">

Noticed some slight spacing differences between each of the `KeywordList` components - maybe would be easiest to flatten all of the `categorisedKeywords` and use a single `KeywordList`

Reply:
Good catch - in the end removed the `useMemo` altogether as we're not giving any indication of category, just throwing keywords on the screen.

Reply:
Without using `getKeywordsForCategories` we aren't filtering to only include:
```
      'Molecular function',
      'Biological process',
      'Disease',
```
but will have all:
```
export type KeywordCategory =
  | 'Biological process'
  | 'Cellular component'
  | 'Coding sequence diversity'
  | 'Developmental stage'
  | 'Disease'
  | 'Domain'
  | 'Ligand'
  | 'Molecular function'
  | 'PTM'
  | 'Technical term';
```
Seems we intentionally filtered this in the past - do we still want to do this?

Reply:
Right, good point I thought it was like aspects - I'll revert and flatten.

Reply:
You could also just filter yourself, something like "keywords.filter(kw => selectedCategories.has(kw.category))" with selectedCategories the set of 3 categories, defined outside of the component, and you should be good to go.